### PR TITLE
Add remaining translations

### PR DESCRIPTION
**Pull request by Kegsay**
Every file has now been manually vetted by me. Due to the extent of
the changes, I've been unable to test all scenarios to make sure this
all works. :(

The main concerns I have are:
 - Missing import for `_t` or `_tJsx`.
 - Stupid bracing mismatches.
 - Incorrect placeholder usage (not doing `%(var)s`, missing the `s` or using `{}` instead of `()`, etc).

So if the code reviewer can please pay particular attention to that, it would be appreciated. My mind is fried so having a 2nd pair of eyes will help.

### CODE REVIEW

File: src/components/views/rooms/AuxPanel.js [hunk 1]
```diff
@@ -95,9 +95,14 @@ module.exports = React.createClass({
             }
             else {
                 joinText = (<span>
-                    Join as <a onClick={(event)=>{ this.onConferenceNotificationClick(event, 'voice');}}
-                               href="#">voice</a> or <a onClick={(event)=>{ this.onConferenceNotificationClick(event, 'video'); }}
-                               href="#">video</a>.
+                    {_tJsx(
+                        "Join as <a>voice</a> or <a>video</a>.",
```
Comment (ara4n) [inline src/components/views/rooms/AuxPanel.js]: they might want to change the ordering of the strings in the string, plus it's confusing to use the same pattern for both.  I'd use `<a type="voice">` or something similar.

File: src/components/views/settings/ChangePassword.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ module.exports = React.createClass({
             onCheckPassword: function(oldPass, newPass, confirmPass) {
                 if (newPass !== confirmPass) {
                     return {
-                        error: _t("New passwords don't match") + "."
```
Comment (Kegsay) [inline src/components/views/settings/ChangePassword.js]: We consistently don't put a `.` on other errors in this component, so it's easier to remove this one.

File: src/components/views/settings/DevicesPanel.js [hunk 3]
```diff
@@ -54,10 +55,10 @@ export default class DevicesPanel extends React.Component {
                 var errtxt;
                 if (error.httpStatus == 404) {
                     // 404 probably means the HS doesn't yet support the API.
-                    errtxt = "Your home server does not support device management.";
+                    errtxt = _t("Your home server does not support device management.");
                 } else {
                     console.error("Error loading devices:", error);
-                    errtxt = "Unable to load device list.";
```
Comment (Kegsay) [inline src/components/views/settings/DevicesPanel.js]: Translation for the non `.` version already exists.

File: src/components/views/rooms/RoomPreviewBar.js [hunk 4]
```diff
@@ -124,58 +123,75 @@ module.exports = React.createClass({
...
-            // TODO: find a way to respect HTML in counterpart!
             joinBlock = (
                 <div>
                     <div className="mx_RoomPreviewBar_invite_text">
                         { _t('You have been invited to join this room by %(inviterName)s', {inviterName: this.props.inviterName}) }
                     </div>
                     <div className="mx_RoomPreviewBar_join_text">
-                        { _t('Would you like to') } <a onClick={ this.props.onJoinClick }>{ _t('accept') }</a> { _t('or') } <a onClick={ this.props.onRejectClick }>{ _t('decline') }</a> { _t('this invitation?') }
+                        { _tJsx(
+                            'Would you like to <a>accept</a> or <a>decline</a> this invitation?',
```
Comment (ara4n) [inline src/components/views/rooms/RoomPreviewBar.js]: similarly, we need to support switching the order.

File: src/i18n/strings/en_EN.json [hunk 5]
```diff
@@ -120,17 +120,21 @@
   "zh-tw":"Chinese (Taiwan)",
   "zu":"Zulu",
   "A registered account is required for this action": "A registered account is required for this action",
+  "a room": "a room",
   "A text message has been sent to +%(msisdn)s. Please enter the verification code it contains": "A text message has been sent to +%(msisdn)s. Please enter the verification code it contains",
-  "accept": "accept",
```
Comment (Kegsay) [inline src/i18n/strings/en_EN.json]: This originally formed part of the sentence "would you like to accept or decline" which we now use in the full JSX form. Independently from that, I needed to add in Accept/Decline buttons, hence the ultimate tense change here.

File: src/components/views/rooms/RoomSettings.js [hunk 6]
```diff
@@ -754,7 +754,11 @@ module.exports = React.createClass({
         if (this.state.join_rule === "public" && aliasCount == 0) {
             addressWarning =
                 <div className="mx_RoomSettings_warning">
-                		{ _t('To link to a room it must have') } <a href="#addresses"> { _t('an address') }</a>.
+                		{ _tJsx(
+                            'To link to a room it must have <a>an address</a>.',
```
Comment (ara4n) [inline src/components/views/rooms/RoomSettings.js]: broken indentation, i think?

File: src/i18n/strings/en_EN.json [hunk 7]
```diff
@@ -197,13 +201,14 @@
   "Claimed Ed25519 fingerprint key": "Claimed Ed25519 fingerprint key",
   "Clear Cache and Reload": "Clear Cache and Reload",
   "Clear Cache": "Clear Cache",
-  "Click here": "Click here",
```
Comment (Kegsay) [inline src/i18n/strings/en_EN.json]: For all cases where I have removed strings, I have manually grepped both react-sdk/riot-web to make sure it was only used in the 1 place and was safe to remove.

**Comment by ara4n** [comment]
have read through the whole thing; lgtm other than the two tJsx comments.

File: src/components/views/rooms/RoomSettings.js (hunk 6 above)
Comment (Kegsay) [inline src/components/views/rooms/RoomSettings.js]: Done. https://github.com/matrix-org/matrix-react-sdk/pull/1056/commits/f2efdf8a4f4fbf0122dbd9aa266bde4ed713ccd5

File: src/components/views/rooms/RoomPreviewBar.js (hunk 4 above)
Comment (Kegsay) [inline src/components/views/rooms/RoomPreviewBar.js]: Done. https://github.com/matrix-org/matrix-react-sdk/pull/1056/commits/f2efdf8a4f4fbf0122dbd9aa266bde4ed713ccd5

File: src/components/views/rooms/AuxPanel.js (hunk 1 above)
Comment (Kegsay) [inline src/components/views/rooms/AuxPanel.js]: Done. https://github.com/matrix-org/matrix-react-sdk/pull/1056/commits/f2efdf8a4f4fbf0122dbd9aa266bde4ed713ccd5

**Comment by ara4n** [comment]
lgtm, thanks!

**Pull request merged**